ctdcal/process_ctd.py

```diff
@@ -975,52 +975,56 @@ def process_reft(ssscc_list, reft_dir="data/reft/"):
                 print("refT file for cast " + ssscc + " does not exist... skipping")
                 return
 
-def load_reft_data(reft_file,index_name = 'btl_fire_num'):
-    """ Loads reft_file to dataframe and reindexes to match bottle data dataframe"""
-
-    # loading in REFTMP_FLAG_W here will conflict with the REFTMP_FLAG_W determined during temperature calibration
-    #reft_data = pd.read_csv(reft_file,usecols=['btl_fire_num','T90','REFTMP_FLAG_W'])
+def _load_reft_data(reft_file, index_name="btl_fire_num"):
+    """
+    Loads reft_file to dataframe and reindexes to match bottle data dataframe
 
-    reft_data = pd.read_csv(reft_file,usecols=['btl_fire_num','T90'])
+    Note: loading in REFTMP_FLAG_W here will conflict with the REFTMP_FLAG_W
+    determined during temperature calibration.
+    """
+    reft_data = pd.read_csv(reft_file, usecols=["btl_fire_num","T90"])
     reft_data.set_index(index_name)
-
-    reft_data['SSSCC_TEMP'] = reft_file[-14:-9]
+    reft_data['SSSCC_TEMP'] = Path(reft_file).stem.split("_")[0]
     reft_data['REFTMP'] = reft_data['T90']
 
     return reft_data
 
-def load_salt_data(salt_file, index_name= 'SAMPNO'):
-    salt_data = pd.read_csv(salt_file,usecols = ['SAMPNO','SALNTY','BathTEMP','CRavg'])
+def _load_salt_data(salt_file, index_name="SAMPNO"):
+    """
+    Loads salt_file to dataframe and reindexes to match bottle data dataframe
+    """
+    salt_data = pd.read_csv(
+        salt_file, usecols=["SAMPNO", "SALNTY", "BathTEMP", "CRavg"]
+    )
     salt_data.set_index(index_name)
-    salt_data['SSSCC_SALT'] = salt_file[-15:-10]
-    salt_data.rename(columns={'SAMPNO':'SAMPNO_SALT'}, inplace=True)
-    return salt_data
-
+    salt_data["SSSCC_SALT"] = Path(salt_file).stem.split("_")[0]
+    salt_data.rename(columns={"SAMPNO": "SAMPNO_SALT"}, inplace=True)
 
+    return salt_data
 
-def load_btl_data(btl_file,cols=None):
 
-    """ex. '/Users/k3jackson/p06e/data/bottle/00201_btl_mean.pkl'"""
 
+def _load_btl_data(btl_file, cols=None):
+    """
+    Loads "bottle mean" CTD data from .pkl file. Function will return all data unless
+    cols is specified (as a list of column names)
+    """
     btl_data = dataToNDarray(btl_file,float,True,',',0)
-
     btl_data = pd.DataFrame.from_records(btl_data)
     if cols != None:
         btl_data = btl_data[cols]
-
-    ssscc = btl_file[-18:-13]
-
-    btl_data['SSSCC'] = ssscc
+    btl_data["SSSCC"] = Path(btl_file).stem.split("_")[0]
 
     return btl_data
 
 
-def load_time_data(time_file):
+# MK: deprecated 04/28/20, use load_all_ctd_files
```

Comment:
We should either make a policy of dropping all deprecated code 1 year later or at X version number or put a proposed drop date.